web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -338,6 +351,8 @@ function ProjectDetails(): JSX.Element {
       projectBySlug?.data.projectBySlug?.published;
   const projectLocation = projectMetadata?.['schema:location'];
 
+  const isTerrasos = normalizedProject.type === 'TerrasosProjectInfo';
```

Comment:
can we use the `IS_TERRASOS` constant from `lib/env` in this case?

Reply:
no this isn't the same thing
IS_TERRASOS just means it's the terrasos app, which allows us to use their custom theme and features as on https://terrasos.app.regen.network/
(basically it's just `VITE_MARKETPLACE_CLIENT` env var that we need to set to tell the app to run as regen marketplace or terrasos app)
whereas here, we're checking whether a particular project is a terrasos project or not, and we checj the `@type` value that is stored in the project metadata